doc/classes/SceneTree.xml

```diff
@@ -137,6 +137,7 @@
 			<param index="0" name="name" type="StringName" />
 			<description>
 				Returns [code]true[/code] if the given group exists.
+				A group exists if any [Node] belongs to it (see [method Node.add_to_group]). Groups without nodes are removed automatically.
```

Comment:
```suggestion
				A group exists if any [Node] in the tree belongs to it (see [method Node.add_to_group]). Groups without nodes are removed automatically.
```
Nodes outside the tree can have groups, but don't count here, nitpick but might be clearer

Reply:
Might even suggest merging the two lines into something like:
"Returns true if there any [Node]s in the tree belonging to this group..."